server/persistence/dynamo/utils.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	dbName       = `cribbage`
-	partitionKey = `DDBid`
+	partitionKey = `gamesV1`
```

Comment:
📖 Is it a good idea to name our partition which stores games, players, interactions, etc. `gamesV1`?

Reply:
true. I guess it should be descriptive of what's in the value attribute field. Perhaps `uuid` or simply `id`? I like something that's "unique enough" so it's easy to correlate and search for in the code. Maybe `cribbageID`?

Reply:
`cribbageID` sounds good to me. I suppose this is one of the idiosyncrasies of dynamo preferring one table for storing many types of things :shrug: